src/constraints.js

```diff
@@ -0,0 +1,135 @@
+import { _ } from 'underscore'
+import utilities from './utilities'
+const moment = require('moment')
```

Comment:
moment lib does not allow somehow to use import. But I need to check, if the new version of moment will allow it